File: mDMP/app/src/main/java/com/example/mdmp/Product.java
package com.example.mdmp;

public class Product {

    private String name;
    private String gramms;
    private String protein;
    private String fats;
    private String carbs;
    private String calories;

    public String getName(){
        return name;
    }
    public String getGramms(){
        return gramms;
    }
    public String getProtein(){
        return protein;
    }
    public String getFats(){
        return fats;
    }
    public String getCarbs(){
        return carbs;
    }
    public String getCalories(){
        return calories;
    }

    public void setName(String name) {
        this.name = name;
    }
    public void setGramms(String gramms) {
        this.gramms = gramms;
    }
    public void setProtein(String protein) {
        this.protein = protein;
    }
    public void setFats(String fats) {
        this.fats = fats;
    }
    public void setCarbs(String carbs) {
        this.carbs = carbs;
    }
    public void setCalories(String calories) {
        this.calories = calories;
    }
}

File: mDMP_Sirvice/Business_Logic_Layer/Product.cs
using Business_Logic_Layer.Interface;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Business_Logic_Layer
{
    public class Product 
    {
        public string Name { get; set; }
        public double Gramms { get; set; }
        public double Protein { get; set; }
        public double Fats { get; set; }
        public double Carbs { get; set; }
        public double Calories { get; set; }

    }
}
